libraries/AP_CANManager/AP_CANDriver.cpp

```diff
@@ -51,6 +51,14 @@ const AP_Param::GroupInfo AP_CANManager::CANDriver_Params::var_info[] = {
     AP_SUBGROUPPTR(_piccolocan, "PC_", 5, AP_CANManager::CANDriver_Params, AP_PiccoloCAN),
 #endif
 
+    // @Param: PROTOCOL2
+    // @DisplayName: Secondary protocol with 11 bit CAN addressing
+    // @Description: Secondary protocol with 11 bit CAN addressing
```

Comment:
Maybe we should add a comment to say that PROTOCOL should be set to DroneCAN for this to work.